Dockerfile

```
@@ -28,6 +28,24 @@ WORKDIR /app
 # Copy binary from builder (make build outputs to bin/)
 COPY --from=builder /build/bin/hyperfleet-adapter /app/adapter
 
+<<<<<<< HEAD
```

Hadolint (2.14.0)

[error] unexpected '<'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)